res/print_makepkg_config.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/bash
+
+source "${LIBRARY:-/usr/share/makepkg}/util/config.sh";
```

Comment:
This is apparently public, as per [this `makepkg` commit message](https://git.archlinux.org/pacman.git/commit/scripts/libmakepkg/util/config.sh.in?id=a00615bfdad628299352b94e0f44d211a758fd17)

Reply:
Can you please add this as a comment in the script as well, for future reference?

Reply:
Good point! :+1: